apollo-core/src/main/java/com/ctrip/framework/apollo/core/utils/ClassLoaderUtil.java

```diff
@@ -37,7 +37,6 @@
       }
     } catch (Throwable ex) {
       classPath = System.getProperty("user.dir");
-      ex.printStackTrace();
```

Comment:
how about "logger.warn("Failed to locate class path, fallback to user.dir: {}", classPath, ex)"?